spec/lib/jll_property_valuation/api_spec.rb

```diff
@@ -1,951 +1,823 @@
 require "spec_helper"
 
 RSpec.describe JllPropertyValuation::Api do
-  let(:api_key) { TODO: <your token here> }
+  let(:api_key) { TODO: Your API Key here }
   let(:subject) {
     JllPropertyValuation::Api.new(
-      username: TODO: <your username here>,
-      password: TODO: <your password here>,
-      api_key: api_key
+      TODO: Username here,
+      TODO: Password here,
+      api_key
     )
   }
 
+  describe '#auth_ping' do
+    it "should return a hash with correct keys" do
+      expect(subject.auth_ping().keys).to include("Status", "Message", "Data")
+    end
+  end
+
+  describe "#auth_login" do
+    it "returns the correct keys" do
+      expect(subject.auth_login().keys).to include("Status", "Message", "Data")
+    end
+
+    it "returns the correct status" do
```

Comment:
this is not a test.

A test would be to make sure it returns ok status with an ok login, and not when it's a bad login